src/sonic-build-hooks/hooks/dpkg

```diff
@@ -3,7 +3,10 @@
 . /usr/local/share/buildinfo/scripts/buildinfo_base.sh
 REAL_COMMAND=$(get_command dpkg)
 COMMAND_INFO="Locked by command: $REAL_COMMAND $@"
-lock_result=$(acquire_apt_installation_lock "$COMMAND_INFO" )
+if ! printenv LOCKED;then
```

Comment:
Would it be better/more robust to check here that `LOCKED` actually contains `y`?

Reply:
Added.